sf-pcap.c

```diff
@@ -821,23 +805,19 @@ pcap_dump_open(pcap_t *p, const char *fname)
 		    "A null pointer was supplied as the file name");
 		return NULL;
 	}
-	if (fname[0] == '-' && fname[1] == '\0') {
```

Comment:
If *all* plugins are supposed to treat "-" as meaning "write to the standard output", `pcap_setup_dump()` should probably do so as well, and report that a write error occurred to "(the) standard output" rather than to "-", as the code currently does.